drivers/usb/device/usb_dc_mcux.c

```diff
@@ -34,6 +38,12 @@ static void usb_isr_handler(void);
 #define SETUP_DATA_STAGE_DONE	(0)
 #define SETUP_DATA_STAGE_IN	(1)
 #define SETUP_DATA_STAGE_OUT	(2)
+#define USBD_MCUX_THREAD_STACK_SIZE	(1024)
+
+/* endpoint related macros */
+#define EP0_MAX_PACKET_SIZE	(64)
```

Comment:
Please use USB_CONTROL_EP_MPS from include/zephyr/usb/usb_ch9.h. btw () do not make any sense here.

Reply:
Thanks for pointing this, I have made this change.